### ISSUE TITLE
Up to date checking in build

### ISSUE THREAD

**Issue by pavelhritonenko** [Microsoft/visualfsharp#797]
When last changed item in project is app.config Visual Studio rebuilds that project on every build.

Steps to reproduce (Visual Studio 2015 Update 1):

1. Create new console F# application
2. Build it
3. Build it again, ensure application is up to date
4. Change app.config (add empty line for example)
5. Build solution
6. Build it again, note that application rebuilds every time.

**Comment by KevinRansom** [comment]
Yes, I can see that ... thanks for the bug report.

**Comment by forki** [comment]
@pavelhritonenko does the same thing happen in C#?

**Comment by KevinRansom** [comment]
@forki yes it looks to me like it does.

**Comment by vasily-kirichenko** [comment]
Another (related?) strange behavior:
```diff
-    <None Include="app.config" />
+    <None Include="app.config">
+      <SubType>Designer</SubType>
+    </None>
```

**Comment by forki** [comment]
Ok so the bug is more general. I assumed this since Vasily's PR didn't
contain any IFs at all.

**Comment by enricosada** [comment]
i think this was already fixed ( #11 ), see my comment in https://github.com/Microsoft/visualfsharp/pull/374#issuecomment-99557084

let me check for regression, but i think is by design the behaviour of c#

**Comment by enricosada** [comment]
also #388 ( content in up to date cause rebuild )
